plugins/source/slack/docs/tables/slack_conversation_bookmarks.md

```diff
@@ -0,0 +1,33 @@
+# Table: slack_conversation_bookmarks
+
+https://api.slack.com/methods/bookmarks.list
+
+The composite primary key for this table is (**team_id**, **id**, **channel_id**).
+
+## Relations
+This table depends on [slack_conversations](slack_conversations.md).
+
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|team_id (PK)|String|
+|id (PK)|String|
+|channel_id (PK)|String|
+|title|String|
+|link|String|
+|emoji|String|
+|icon_url|String|
+|type|String|
+|date_created|Int|
```

Comment:
shoult be `schema.TypeTimestamp`, use type transformer for this

Reply:
Good call